tests/plugins/tabletools/cellproperties.js

```diff
@@ -129,6 +129,64 @@
 				dialog.setValueOf( 'info', 'width', 10 );
 				dialog.setValueOf( 'info', 'widthType', 'px' );
 			} );
+		},
+
+		'test test': function() {
+			bender.editorBot.create( {
+				name: 'required',
+				config: {
+					allowedContent: {
```

Comment:
I am not sure, but maybe it can be replaced with something like:
`allowedContent: *[lang,dir]; table tbody tr...`

Test with `disallowedContent` rule should be added too.